### PR TITLE
migrate log in pkg/volume/plugins.go

### PR DESCRIPTION
**Pull request by huchengze**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Ref:

* [keps/sig-instrumentation/1602-structured-logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging)
* [Structured Logging migration instructions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

**Does this PR introduce a user-facing change?**:

```release-note
Migrate some log messages to structured logging in pkg/volume/plugins.go.
```

### CODE REVIEW

File: pkg/volume/plugins.go [hunk 1]
```diff
@@ -631,12 +631,12 @@ func (pm *VolumePluginMgr) InitPlugins(plugins []VolumePlugin, prober DynamicPlu
 		}
 		err := plugin.Init(host)
 		if err != nil {
-			klog.Errorf("Failed to load volume plugin %s, error: %s", name, err.Error())
+			klog.ErrorS(err, "Failed to load volume plugin", "name", name)
```
Comment (shubheksha) [inline pkg/volume/plugins.go:634]:
```suggestion
			klog.ErrorS(err, "Failed to load volume plugin", "pluginName", name)
```

File: pkg/volume/plugins.go [hunk 2]
```diff
@@ -631,12 +631,12 @@ func (pm *VolumePluginMgr) InitPlugins(plugins []VolumePlugin, prober DynamicPlu
...
 			allErrs = append(allErrs, err)
 			continue
 		}
 		pm.plugins[name] = plugin
-		klog.V(1).Infof("Loaded volume plugin %q", name)
+		klog.V(1).InfoS("Loaded volume plugin", "name", name)
```
Comment (shubheksha) [inline pkg/volume/plugins.go:639]: Same as above

File: pkg/volume/plugins.go [hunk 3]
```diff
@@ -652,7 +652,7 @@ func (pm *VolumePluginMgr) initProbedPlugin(probedPlugin VolumePlugin) error {
 		return fmt.Errorf("Failed to load volume plugin %s, error: %s", name, err.Error())
 	}
 
-	klog.V(1).Infof("Loaded volume plugin %q", name)
+	klog.V(1).InfoS("Loaded volume plugin", "name", name)
```
Comment (shubheksha) [inline pkg/volume/plugins.go:655]: Same as above

File: pkg/volume/plugins.go [hunk 4]
```diff
@@ -956,10 +956,10 @@ func (pm *VolumePluginMgr) FindExpandablePluginBySpec(spec *Spec) (ExpandableVol
 		if spec.IsKubeletExpandable() {
 			// for kubelet expandable volumes, return a noop plugin that
 			// returns success for expand on the controller
-			klog.V(4).Infof("FindExpandablePluginBySpec(%s) -> returning noopExpandableVolumePluginInstance", spec.Name())
+			klog.V(4).InfoS("FindExpandablePluginBySpec -> returning noopExpandableVolumePluginInstance", "spec.Name()", spec.Name())
```
Comment (shubheksha) [inline pkg/volume/plugins.go:959]:
```suggestion
			klog.V(4).InfoS("FindExpandablePluginBySpec -> returning noopExpandableVolumePluginInstance", "specName", spec.Name())
```

File: pkg/volume/plugins.go [hunk 5]
```diff
@@ -956,10 +956,10 @@ func (pm *VolumePluginMgr) FindExpandablePluginBySpec(spec *Spec) (ExpandableVol
...
 			return &noopExpandableVolumePluginInstance{spec}, nil
 		}
-		klog.V(4).Infof("FindExpandablePluginBySpec(%s) -> err:%v", spec.Name(), err)
+		klog.V(4).InfoS("FindExpandablePluginBySpec -> err", "spec.Name()", spec.Name(), "err", err)
```
Comment (shubheksha) [inline pkg/volume/plugins.go:962]: Same as above

**Review by shubheksha — CHANGES_REQUESTED** [review]

File: pkg/volume/plugins.go (hunk 4 above)
Comment (serathius) [reply pkg/volume/plugins.go:959]: What is the `spec` here? I assume spec is short for specification, but of what a plugin, volume etc?

**Pull request closed without merging**